content/tutorials/get-started/set-up-secure-signing.md

```diff
@@ -0,0 +1,134 @@
+# Set Up Secure Signing
+
+To submit transactions to the XRP Ledger, you need a way to digitally sign them without compromising the security of your secret keys. (If others gain access to your secret keys, they have as much control over your accounts as you do, and can steal or destroy all your money.) This page summarizes how to set up such an environment so you can sign transactions securely.
+
+**Tip:** If you are not submitting transactions to the network, you can safely use a trustworthy public server, such as the ones run by Ripple, to monitor for incoming transactions or just to read other network activity. All transactions, balances, and data in the XRP Ledger itself is public.
+
+There are several configurations with varying levels of security that may be acceptable for your situation. Choose one of the following that best fits your needs:
+
+- [Run `rippled` locally](#run-rippled-locally), or [in the same LAN](#run-rippled-on-the-same-lan).
+- [Use a client library](#use-a-client-library-with-local-signing) that can do local signing.
+- [Use a secure VPN to connect to a remote `rippled` machine](#use-a-secure-vpn-with-a-remote-rippled-server) you trust.
+
+<!-- Source for all diagrams in this article: https://docs.google.com/presentation/d/1BfGyWgC0njoPiKUZz3gXHMVSUINE3Q-_lHqY_D0TGwg/ -->
+
+## Insecure Configurations
+
+[![Diagram of insecure configurations](img/insecure-signing-options.png)](img/insecure-signing-options.png)
+
+Any configuration in which outside sources may gain access to your secret key is dangerous, and is likely to result in a malicious user stealing all your XRP. Examples of such configurations include ones where you use the [sign method][] of someone else's `rippled` server over the internet, or you transmit your secret key in plain text over the internet to your own server.
```

Comment:
A malicious user could do more than than steal all your XRP. If you had IOUs they may steal those, etc. Consider rewording to suggest that the malicious user would gain access to the account and be able to authorize the execution of transactions.